sofiykam.md

```diff
@@ -0,0 +1,37 @@
+both of which are very important parts of the career I want to pursue, and also are big hobbies and passions that I have. 
+
+
+## First I will purchase all of the materials needed to create the predictor. 
+When I have all of the materials, then I will actually prepare for set-up by connecting the SD card to the laptop, 
+and downloading all of the necessary information. Then I will set up the Jetson Kit and attach all the necessary 
+cables and pieces to the Jetson.(the jetson and pulse oximeter + camera will be connected with the ethernet cable and the USB plugs they have included.)
+I will need to turn on the power supply and wait for the system to reboot, and then conduct 
+all the initial setup steps for the Jetson. Then I will download the Jetson interface and program it to run when asked to, 
+I will use the datasets to give the Jetson information to analyze (or use a camera to provide live videos/photos), 
+and finally continue training the Jetson and troubleshooting until it works more and more efficiently. I will 
+also need to practice running the program many times to prevent failures and incorrect measurements. In addition, the webcam 
+will be used to analyze the skin and predict what is happening to the skin of the patient, 
+and the pulse oximiter will be used to find data that can also be analyzed by the program. 
+
+## Budget
+
+| Product         | Supplier/Link                         | Cost   |
+| --------------- | ------------------------------------- | ------ |
+| Type C power supply   | https://www.amazon.com/dp/B08R7N4CP6/ref=sspa_dk_detail_3psc=1&pd_rd_i=B08R7N4CP6&pd_rd_w=MF4j6&contentid=amzn1.sym.dd2c6db76626-466d-bf04-9570e69a7df0&pf_rd_p=dd2c6db7-6626-466d-bf04-9570e69a7df0&pf_rd_r=N1XSQFM38F0KSV7X9CQ7&pd_rd_wg=x2yBP&pd_rd_r=27a3942d-ac16-4539-88205b34b0ac4265&s=pc&sp_csd=d2lkZ2V0TmFtZT1zcF9kZXRhaWxfdGhlbWF0aWM&spLa=ZW5jcnlwdGVkUXVhbGlmaWVyPUEyRjNTN1pENDc1UDRJJmVuY3J5cHRlZElkPUEwMTA3OTA5MjJOQ01QRUNTQk45RyZlbmNyeXB0ZWRBZElkPUEwMjg0MjM4MVY3TjJMNlA4V1NIQSZ3aWRnZXROYW1lPXNwX2RldGFpbF90aGVtYXRpYyZhY3Rpb249Y2xpY2tSZWRpcmVjdCZkb05vdExvZ0NsaWNrPXRydWU | $8.59  |
+| NVIDIA Jetson nano developer kit | https://www.amazon.com/NVIDIA-Jetson-Nano-Developer-945-13450-0000-100/dp/B084DSDDLT/ref=sr_1_2crid=35M189S5Q9LT3&keywords=nvidia+jetson+nano+developer+kit+2gb&qid=1672350344&sprefix=nvidia+jetson+nano+developer+kit+2gb%2Caps%2C133&sr=8-2  | $149.00 |
+| Ethernet Cable |https://www.amazon.com/AmazonBasics-RJ45-Cat-6-Ethernet-Patch-Cable-25-Feet-7-6-Meters/dp/B00N2VIWPY/ref=sr_1_3?crid=3G4N4R2JV0HAO&keywords=ethernet%2Bcable&qid=1672351370&sprefix=ethernet%2Bcable%2Caps%2C109&sr=8-3&th=1| $6.50|
+| Camera (Logitech C270 HD WEBCAM) | https://www.amazon.com/Logitech-Desktop-Widescreen-Calling-Recording/dp/B004FHO5Y6/ref=sr_1_1?crid=29DTQJIO6B0YV&keywords=Camera%2B(Logitech%2BC270%2BHD%2BWEBCAM)&qid=1672351499&sprefix=camera%2Blogitech%2Bc270%2Bhd%2Bwebcam%2B%2Caps%2C103&sr=8-1&th=1| $24.99|
```

Comment:
how do you know that this camera is compatible with the jetson nano?

Reply:
There is a specific tutorial that I'm following, and the person that made it already made sure it works. (its the same camera they are using)

Reply:
ooo cool! would you be able to link the tutorial within your file?